sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/MeterSdk.java

```diff
@@ -112,8 +112,11 @@ public BatchRecorderSdk newBatchRecorder(String... keyValuePairs) {
     return result;
   }
 
-  /** Creates a {@link Batcher}, by using the {@link ViewRegistry} to do the actual work. */
-  Batcher createBatcher(
+  /**
+   * Creates a {@link InstrumentAccumulator}, by using the {@link ViewRegistry} to do the actual
+   * work.
+   */
+  InstrumentAccumulator createBatcher(
```

Comment:
nit: this method name would probably need to be updated to `createAccumulator`

Reply:
It will be moved and renamed in a followup.